benchmarks/h3.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const { createServer } = require('http')
+const { createApp, toNodeListener, eventHandler } = require('h3')
+
+const app = createApp()
+app.use('/', eventHandler(() => ({ hello: 'world' })))
```

Comment:
as @pi0 mentioned before:
> The router is fast skipped for this case. Is it still eligible to set hasRouter flag or are there any baselines a router should handle?

Reply:
Well, it should act as a router. If it transposes `http://localhost:3000/` to this handler and only this handler. It's valid.

Reply:
It "kind of" does.

While you can also opt-in for a "real router", the naive version just checks if the URL starts with the given string. So `/abc/` would match `/abc/` but also `/abc/ajsdioasdajo`. `/` would match *every* URL here.

Reply:
So, technically it shouldn't be eligible to set the `hasRouter` flag. Once it clearly has advantages in comparison to routers.

Reply:
Makes sense 👍